src/Tizen.Nlp/Tizen.Nlp/NaturalLanguageProcess.cs

```diff
@@ -60,8 +62,13 @@ public class NaturalLanguageProcess
         /// An construct method  to init local env of NLP .
         /// </summary>
         /// <since_tizen> 5 </since_tizen>
+        /// <feature>
+        /// http://tizen.org/feature/nlp
+        /// </feature>
+        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
         public NaturalLanguageProcess()
         {
+            ValidateFeatureSupported(FeatureNlp);
```

Comment:
If you use the dummy assembly created automatically, you don't need check itself.
Dummy assembly will throw NotSupportedException when access all methods of this assembly.

Reply:
Fixed